DependencyInjection/HttplugExtension.php

```diff
@@ -27,6 +27,13 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('services.xml');
         $loader->load('plugins.xml');
         $loader->load('discovery.xml');
+        if ($config['toolbar']['enabled'] && $container->hasParameter('kernel.debug') && $container->getParameter('kernel.debug')) {
```

Comment:
You are correct. I should redo this. 
What I really want to know is if the service `profiler` exits and if its configuration for `toolbar` is true. 

I'll investigate some more.